src/components/CountryDetails.js

```diff
@@ -0,0 +1,61 @@
+import { useState, useEffect} from 'react';
+import { useParams } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
+import React from 'react';
+
+function CountryDetails (props) {
+
+  const [details, setDetails] = useState({});
+
+  const {countryId} = useParams();
+
+/*   Not sure why my page doesn't show anything at all on the browser. I have tried to apply the logic anyway to display the borders, but I have a feeling that my code is all over the place, sorry! */
+
+  useEffect(() => {
+      /* I should maybe have this in there somewhere? : country.alpha3Code === countryId; 
+    }); */
+    /* and something like this? : setDetails(prevValue) */
+  }, [countryId])
+
+    return (
+        <div class="col-7">
+        <h1>CountryDetails</h1>
+            <h1>{details.name}</h1>
+            <table class="table">
+              <thead></thead>
+              <tbody>
+                <tr>
+                  <td style="width: 30%">Capital</td>
```

Comment:
styles in jsx, you do like this:
                  <td style = {{width: "30%"}}>Capital</td>